File: pet-sitting-backend/src/main/java/fr/aston/petSitting/modele/BookingModel.java
package fr.aston.petSitting.modele;
import java.math.BigDecimal;
import java.util.Date;

import fr.aston.petSitting.entities.AnimalGenderEnum;
import fr.aston.petSitting.entities.AnimalTypeEnum;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;


@Data
@NoArgsConstructor
@AllArgsConstructor
public class BookingModel {
	private int id;

	private BigDecimal totalPrice;
	private Date startDate;
	private Date endDate;
	private int animalId;
	private int serviceId;

}

File: pet-sitting-front/libs/services/src/lib/services/models/booking.model.ts
import { BookingStatuEnum } from "./statusBooking.enum";

export interface BookingModel {
  id?: number;
  startDate?: Date;
  endDate?: Date;
  totalPrice?:number;
  animalId?:number;
  serviceId?:number;
  status:BookingStatuEnum;
}
